ProjFS.Mac/PrjFSKext/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -144,7 +144,7 @@ VirtualizationRootHandle VirtualizationRoot_FindForVnode(vnode_t vnode, const Fs
     {
         ProfileSample iterationSample(Probe_VirtualizationRoot_FindIteration);
 
-        rootHandle = FindOrDetectRootAtVnode(vnode, nullptr /* vfs context */, vnodeFsidInode);
+        rootHandle = FindOrDetectRootAtVnode(vnode, vfsContext, vnodeFsidInode);
```

Comment:
Is this the actual bug fix? I can't tell why this matters though, because even though we're passing the vfsContext in here, it's not being used anywhere inside `FindOrDetectRootAtVnode`. Can we actually remove this parameter?

Reply:
Note: `FindOrDetectRootAtVnode` passes the context over to `Vnode_ReadXattr`, but that function ignores it completely. So it looks like we can remove it from both.

Reply:
Huh! You appear to be correct, I somehow missed that. I guess all the NULL vfs_context cases must already have been dealt with previously. Well, that pretty much defeats the point of this PR…

Reply:
I'm in favor of compile-time enforcement wherever we can get it

Reply:
Alright, no true bugfix then, but I've kept the annotations and removed the obsolete parameter instead of passing the context through.